index.js

```diff
@@ -28,29 +29,36 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
 
       console.log('='.repeat(consoleWidth()));
 
+      if(typeof process.argv[4] !== 'undefined'){
+        if(typeof Number(process.argv[4]) === 'number'){
```

Comment:
i also see you're reading *arguments* and not *options* so you'll have to remove this entire `process.argv[4]` and replace it with option

Reply:
Should this logic sit inside something like 

```
.option('-r, --results', 'num of results', () => {
    // stuff
});
```
or something?

Reply:
I still have to read through the commander docs

Reply:
@jwu910 don't offer suggestions without reading the docs... it will get confusing haha...
@psyduc and I were looking into the docs and trying out a few things to get the logic to work. I didn't see any examples laid out the way you have it written, only see functions when doing actions. The examples do show this though:
```javascript
...
.option("-s, --setup_mode [mode]", "Which setup mode to use")
  .action(function(env, options){
    var mode = options.setup_mode || "normal";
...
```